examples/nixnet/lin-signal-single-point-output.py

```diff
@@ -114,7 +114,7 @@ def check_for_error(status):
         )
         check_for_error(write_state_response.status)
 
-    print("Values are incremented from 0 to 10 in this example.\n")
+    print("Writing 10 values to LIN Port.\n")
```

Comment:
This should be `LIN interface`

Reply:
yes. interface is the 'word' to use in all examples.

Reply:
Done.